packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js

```diff
@@ -0,0 +1,282 @@
+    const before = moment().toISOString();
+    switch (this.period.name) {
+      case this.TRAFFIC_PERIOD_LIST.lastWeek:
+        after = moment()
+          .subtract(7, 'days')
+          .toISOString();
+        break;
+      case this.TRAFFIC_PERIOD_LIST.last2weeks:
+        after = moment()
+          .subtract(14, 'days')
+          .toISOString();
+        break;
+      default:
+        after = moment()
+          .subtract(1, 'days')
+          .toISOString();
+    }
+    const params = {
+      after,
+      before,
+      subnet: this.subnet,
+    };
+    this.networkSecurityService.getTraffic(params).then(({ data }) => {
+      if (data.message) {
+        this.Alerter.error(
+          this.$translate.instant('network_security_dashboard_events_error'),
+          'network_security_error',
+        );
+        return data;
+      }
+      this.results = data;
+      this.resume(after, before, this.subnet);
+      this.loadGraph();
+      return data;
+    });
+  }
+
+  resume(after, before, subnet) {
+    // Set number of events
+    this.numberOfEvents = 0;
+    this.networkSecurityService
+      .getEvents({
+        after,
+        before,
+        subnets: subnet,
+      })
+      .then(({ data }) => {
+        if (data.events) {
+          this.numberOfEvents = data.events.length;
+        }
+      });
+
+    // Set packets dropped
+    let packetsDropped = 0;
+    this.results.pps.dropped.forEach((packet) => {
+      packetsDropped += parseInt(packet, 10);
+    });
+    let unit = TrafficController.getUnitIndex(packetsDropped);
+    let label = '';
+    if (unit > 0) {
+      label = this.units[unit][0].toUpperCase();
+    }
+    this.packetsDropped = `${TrafficController.formatBits(
+      packetsDropped,
+      unit,
+    )} ${label}`;
+
+    // Set bandwith cleaned
+    let bandwithCleaned = 0;
+    this.results.bps.dropped.forEach((packet) => {
+      bandwithCleaned += parseInt(packet, 10);
+    });
+    unit = TrafficController.getUnitIndex(bandwithCleaned);
+    this.bandwithCleaned = `${TrafficController.formatBits(
+      bandwithCleaned,
+      unit,
+    )} ${this.units[unit].toUpperCase()}`;
+
+    // Set attacks detected
+    this.attacksDetected = 0;
+    this.networkSecurityService
+      .getEvents({
+        after,
+        before,
+        subnets: subnet,
+      })
+      .then(({ data }) => {
+        if (data.events) {
+          data.events.forEach((ev) => {
+            this.attacksDetected += ev.vectors.length;
+          });
+        }
+      });
+  }
+
+  updateStackable(value) {
+    this.isStackable = value;
+    this.loadGraph();
+  }
+
+  loadGraph(isPPs) {
+    if (isPPs !== undefined) {
+      this.isPPs = isPPs;
+    }
+    this.series = [];
+    this.data = [];
+
+    if (this.isStackable) {
+      this.options.scales.yAxes[0].stacked = true;
+    } else {
+      delete this.options.scales.yAxes[0].stacked;
+    }
+
+    this.labels = map(get(this.results, 'timestamps'), (value) => value);
```

Comment:
Try to use native es6
```suggestion
    this.labels = this.results?.timestamps?.map((value) => value)
```